spec/spec_helper.rb

```diff
@@ -15,25 +15,32 @@
 
 Capybara.run_server = false
 
-firefox_options = ::Selenium::WebDriver::Firefox::Options.new
-firefox_options.profile = Selenium::WebDriver::Firefox::Profile.new.tap do |profile|
-  profile['browser.download.dir'] = DownloadHelpers::PATH.to_s
-  # profile["browser.helperApps.neverAsk.openFile"] = "application/x-yaml"
-  profile['browser.download.folderList'] = 2
-  profile['browser.helperApps.neverAsk.saveToDisk'] = 'application/x-yaml'
-end
-
 Capybara.register_driver :my_firefox_driver do |app|
-  Capybara::Selenium::Driver.new(app, browser: :firefox, options: firefox_options)
+  options = Selenium::WebDriver::Firefox::Options.new
+  options.profile = Selenium::WebDriver::Firefox::Profile.new.tap do |profile|
+    profile['browser.download.dir'] = DownloadHelpers::PATH.to_s
+    # profile["browser.helperApps.neverAsk.openFile"] = "application/x-yaml"
```

Comment:
Do we need this comment?

Reply:
Good question!

Reply:
@aaron-collier Looks like it's been there since @jcoyne's initial commit in this repository.